docs/code-quality/fxcop-rule-port-status.md

```diff
@@ -95,6 +102,7 @@ CA1509 | Invalid entry in code metrics rule specification file
 [CA1725](ca1725.md) | Parameter names should match base declaration
 [CA1801](ca1801.md) | Review unused parameters
 [CA1802](ca1802.md) | Use literals where appropriate
+[CA1804](ca1804.md) | Remove unused locals
```

Comment:
`CA1804` was ported, but then removed and deprecated in favor of another rule in IDE CodeStyle NuGet package. See https://github.com/dotnet/roslyn-analyzers/issues/2109 for details.

Reply:
Alright, I'll move it to the bottom of the page then.